packages/server/src/schema/schema.graphql

```diff
@@ -18,6 +18,7 @@ type Mutation {
 
   startTraining(id: ID!): Project
   haltTraining(id: ID!): Project
+  exportCheckpoint(id: ID!, checkpointNumber: Int!, name: String!): Project
```

Comment:
Do we need a field on Project that is the exported path?

Reply:
will eventually be needed, the container does not currently do that.